mvpa2/tests/test_hyperalignment.py

```diff
@@ -171,6 +171,49 @@ def test_basic_functioning(self, ref_ds, zscore_common, zscore_all):
         rerrors = ha.ca.residual_errors.samples
         self.assertEqual(rerrors.shape, (1, n))
 
+    def test_hpal_svd_combo(self):
+        # get rotated dataset for test
+        ds4l = datasets['uni4large']
+        ds_orig = ds4l[:, ds4l.a.nonbogus_features]
+        nf = ds_orig.nfeatures
+        n = 4  # # of datasets to generate
+        dss_rotated_clean = [random_affine_transformation(ds_orig, scale_fac=100, shift_fac=10)
+                             for i in xrange(n)]
+        # zscoring
+        _ = [zscore(sd, chunks_attr=None) for sd in dss_rotated_clean]
+        # Test if it is close to doing hpal+SVD in sequence outside hpal
+        # First, as we do in sequence outside hpal
+        ha = Hyperalignment()
+        mappers_orig = ha(dss_rotated_clean)
+        dss_clean_back = [m.forward(ds_)
+                          for m, ds_ in zip(mappers_orig, dss_rotated_clean)]
+        _ = [zscore(sd, chunks_attr=None) for sd in dss_clean_back]
+        dss_mean = np.mean([sd.samples for sd in dss_clean_back], axis=0)
+        zscore(dss_mean, chunks_attr=None)
+        from mvpa2.mappers.svd import SVDMapper
+        svm = SVDMapper()
+        svm.train(dss_mean)
+        dss_sv = [svm.forward(sd) for sd in dss_clean_back]
+        # Test for SVD dimensionality reduction
+        for output_dim in range(1, 5):
+            ha = Hyperalignment(output_dim=output_dim)
+            ha.train(dss_rotated_clean)
+            mappers = ha(dss_rotated_clean)
+            dss_clean_back = [m.forward(ds_)
+                              for m, ds_ in zip(mappers, dss_rotated_clean)]
+            for sd in dss_clean_back:
+                assert(sd.nfeatures == output_dim)
```

Comment:
here it is now more of a smoke test really... i would have tuned test data do bear needed features -- i.e. generate data only within output_dim, rotate into higher dimensions and see if we recover it back into lower dimensions more or less correctly